test_modules/test_module_1/test_module_1.module

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
**Looks like module hook_help is missing**
Always implement hook_help in newly created custom module